From c682c2457bafcf35671ae78328414f111332de60 Mon Sep 17 00:00:00 2001
From: Markus Stenberg <markus.stenberg@iki.fi>
Date: Thu, 22 May 2014 17:01:55 +0300
Subject: [PATCH] UML+NetKit kernel args can be (at least) 6 lines long => 1kb
 is hopefully safe.

---
 initd/init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/initd/init.c b/initd/init.c
index d458f29..cd01540 100644
--- a/initd/init.c
+++ b/initd/init.c
@@ -53,7 +53,7 @@ static struct sigaction sa_shutdown = {
 static void
 cmdline(void)
 {
-	char line[256];
+	char line[1024];
 	int r, fd = open("/proc/cmdline", O_RDONLY);
 	regex_t pat_cmdline;
 	regmatch_t matches[2];
-- 
1.8.5.2 (Apple Git-48)

